src/BugTracker/project.json

```diff
@@ -6,13 +6,14 @@
     "version": "1.0.0-*",
     "webroot": "wwwroot",
     "dependencies": {
-        "Kestrel": "1.0.0-*",
-        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
-        "Microsoft.AspNet.Mvc": "6.0.0-*",
-        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
-        "Microsoft.AspNet.Diagnostics": "1.0.0-*",
-        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
-        "Microsoft.AspNet.SignalR.Server": "3.0.0-*"
+        "Kestrel": "1.0.0-beta4-*",
```

Comment:
I think you could just make it `*`. When we merge to master we can freeze the versions. That we we need not change the versions for every release.